experimental/e2e/proxy.go

```diff
@@ -121,7 +121,7 @@ func (p *Proxy) append(res *http.Response, ctx *goproxy.ProxyCtx) *http.Response
 		return res
 	}
 	f := p.Fixtures[0]
-	if _, cached := f.Match(res.Request); cached != nil {
```

Comment:
Nit: Renamed `cached` to `matched` or `res` - whatever is most consistent

Reply:
addressed in [0f4efb1](https://github.com/grafana/grafana-plugin-sdk-go/pull/473/commits/0f4efb175948bfb4075dda4a5d3a740611bd6921)